src/components/13-collection/collection--fancy-date.njk

```diff
@@ -0,0 +1,188 @@
+<div class="grid-row padding-8">
+  <div class="mobile-lg:grid-col-12 margin-top-4 mobile-lg:margin-top-0">
+    <h2 class="margin-x-0 margin-y-2">Latest News</h2>
+    <ul class="usa-collection">
+      <li class="usa-collection__item">
+        <a class="usa-collection__calendar-date" href="https://www.performance.gov/presidents-winners-press-release/">
+          <time datetime="2020-09-30T12:00:00+01:00">
+            <span class="usa-collection__calendar-date-month">SEP</span>
+            <span class="usa-collection__calendar-date-day">30</span>
+          </time>
+        </a>
+        <div class="usa-collection__body">
+          <h3 class="usa-collection__heading">
+            <a class="usa-link" href="https://www.performance.gov/presidents-winners-press-release/">Gears of Government President's Award Winners</a>
+          </h3>
+          <p class="usa-collection__teaser">Today, the Administration announces the winners of the Gears of Government President’s Award. This program recognizes the contributions of individuals and teams across the federal workforce who make a profound difference in the lives of the American people.</p>
+          <div class="usa-collection__meta-list">
+            <span class="usa-collection__meta-item">
+              By Performance.gov Team
+            </span>
+            <span class="usa-collection__meta-item">
+              <time datetime="2020-09-30T12:00:00+01:00">
+                September 30, 2020
+              </time>
+            </span>
+          </div>
+
+          <div class="usa-collection__meta-list">
+            <span class="usa-collection__meta-item usa-tag">
```

Comment:
I'm trying to figure out how non-visual experiences will read this markup... is there anything we can do to identify these as tags/labels? https://www.w3.org/WAI/WCAG21/Techniques/html/H49

Reply:
@jonadecker any ideas?

Reply:
I'm wondering if it wouldn't be more appropriate to gather the tags as an unordered list @sknep @wbyoko . The preference always goes to semantic HTML vs. bandaid.

Reply:
DONE